### PR TITLE
docs: Using https://open-vsx.org with thousands of community-supported extensions is not recommended in an air-gapped environment

### PR DESCRIPTION
**Pull request by ibuziuk**
## What does this pull request change?

Screenshot 2024-12-03 at 15 16 46

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by ibuziuk** [comment]
@deerskindoll once merged, please backport to 7.95.x for 3.18

File: modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc [hunk 1]
```diff
@@ -24,8 +24,9 @@ spec:
 ----
 <1> For example: `openVSXURL: "pass:c,a,q[https://open-vsx.org]"`.
 +
-[TIP]
-==== 
+[IMPORTANT]
+====
+* Using link:https://open-vsx.org[https://open-vsx.org] with thousands of community-supported extensions is not recommended in an air-gapped environment, isolated from the internet. In order to reduce the risk of malware infections and unauthorized access to your code use the embedded or self-hosted Open VSX registry with the curated set of extensions.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]:
```suggestion
* Using the extensions from link:https://open-vsx.org[https://open-vsx.org] is not recommended in an air-gapped environment, isolated from the internet. In order to reduce the risk of malware infections and unauthorized access to your code use the embedded or self-hosted Open VSX registry with the curated set of extensions.
```

**Review by deerskindoll — COMMENTED** [review]
a small tweak

File: modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc (hunk 1 above)
Comment (ibuziuk) [reply modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]: tbh, I like the original version more

**Review by svor — APPROVED** [review]
lgtm

File: modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc (hunk 1 above)
Comment (ibuziuk) [reply modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]: potentially, we can remove `with thousands of community-supported extensions` part, but the main point here  is about registry itself (`https://open-vsx.org` in the CR), not individual extentions

**Review by deerskindoll — APPROVED** [review]
lgtm

File: modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc (hunk 1 above)
Comment (ibuziuk) [inline modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc:29]:
```suggestion
* Using link:https://open-vsx.org[https://open-vsx.org] with thousands of community-supported extensions is not recommended in an air-gapped environment, isolated from the internet. In order to reduce the risk of malware infections and unauthorized access to your code use the embedded or self-hosted Open VSX registry with the curated set of extensions.
```

Comment (ibuziuk) [inline modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]:
```suggestion
* Using link:https://open-vsx.org[https://open-vsx.org] is not recommended in an air-gapped environment, isolated from the internet. In order to reduce the risk of malware infections and unauthorized access to your code use the embedded or self-hosted Open VSX registry with the curated set of extensions.
```

Comment (ibuziuk) [reply modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]: @deerskindoll wdyt?

**Pull request merged**